src/SCRIPTS/BF/PAGES/pos_osd.lua

```diff
@@ -0,0 +1,169 @@
+        {["address"] = 66, ["firstVal"] = 143, ["secondVal"] = 144, ["nameCli"] = "osd_sys_goggle_voltage_pos"},
+        {["address"] = 67, ["firstVal"] = 145, ["secondVal"] = 146, ["nameCli"] = "osd_sys_vtx_voltage_pos"},
+        {["address"] = 68, ["firstVal"] = 147, ["secondVal"] = 148, ["nameCli"] = "osd_sys_bitrate_pos"},
+        {["address"] = 69, ["firstVal"] = 149, ["secondVal"] = 150, ["nameCli"] = "osd_sys_delay_pos"},
+        {["address"] = 70, ["firstVal"] = 151, ["secondVal"] = 152, ["nameCli"] = "osd_sys_distance_pos"},
+        {["address"] = 71, ["firstVal"] = 153, ["secondVal"] = 154, ["nameCli"] = "osd_sys_lq_pos"},
+        {["address"] = 72, ["firstVal"] = 155, ["secondVal"] = 156, ["nameCli"] = "osd_sys_goggle_dvr_pos"},
+        {["address"] = 73, ["firstVal"] = 157, ["secondVal"] = 158, ["nameCli"] = "osd_sys_vtx_dvr_pos"},
+        {["address"] = 74, ["firstVal"] = 159, ["secondVal"] = 160, ["nameCli"] = "osd_sys_warnings_pos"},
+        {["address"] = 75, ["firstVal"] = 161, ["secondVal"] = 162, ["nameCli"] = "osd_sys_vtx_temp_pos"},
+        {["address"] = 76, ["firstVal"] = 163, ["secondVal"] = 164, ["nameCli"] = "osd_sys_fan_speed_pos"},
+    }
+    
+if not address then
+    firstVal = items[1]["firstVal"]
+    secondVal = items[1]["secondVal"]
+    address = items[1]["address"]
+    nameCli = items[1]["nameCli"]
+end
+
+x = margin
+y = yMinLim - tableSpacing.header
+
+fields[#fields + 1] = { t = "OSD Element (select)", x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = (#items - 1), vals = { 1 }, upd = function(self) self.updateItems(self) end }
+fields[#fields + 1] = { t = "Screen",               x = -300, y = -300,            sp = x + sp, min = 1, max = 1, vals = { 4 } }
+labels[#labels + 1] = { t = nameCli,                x = x, y = inc.y(lineSpacing), sp = x + sp }
+
+labels[#labels + 1] = { t = "POS",                  x = x,                        y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "OP1",                  x = x + tableSpacing.col,     y = y }
+labels[#labels + 1] = { t = "OP2",                  x = x + tableSpacing.col * 2, y = y }
+labels[#labels + 1] = { t = "OP3",                  x = x + tableSpacing.col * 3, y = y }
+
+fields[#fields + 1] = {                             x = x,                        y = inc.y(tableSpacing.row), min = 0, max = 2047, vals = { 2, 3 } }
+fields[#fields + 1] = {                             x = x + tableSpacing.col,     y = y, min = 0, max = 1, vals = { firstVal, secondVal }, table = { [0] = "OFF", "ON" } }
+fields[#fields + 1] = {                             x = x + tableSpacing.col * 2, y = y, min = 0, max = 1, vals = { firstVal, secondVal }, table = { [0] = "OFF", "ON" } }
+fields[#fields + 1] = {                             x = x + tableSpacing.col * 3, y = y, min = 0, max = 1, vals = { firstVal, secondVal }, table = { [0] = "OFF", "ON" } }
```

Comment:
`vals = { firstVal, secondVal }` will cause these fields to make unintended changes to rssi position. Try selecting any element and change any profile. Without saving, change back to osd_rssi_pos and see.

ALL fields here are making unintended changes to values. The BF lua scripts doesn't really support this at the moment, but we could look into allowing editing field values without having `vals` defined.
For now you could just do `vals = { 1, 2 }` for all fields. This means all fields will make changes to byte 1 and 2 in the received message, but we're not using those values, and we're clearing the whole table before saving anyway.